packages/devextreme/js/__internal/core/license/license_validation_internal.ts

```diff
@@ -0,0 +1,16 @@
+import { Token, TokenKind } from './types';
+
+export function parseLicenseKey(): Token {
+  return {
+    kind: TokenKind.internal,
+    internalUsageId: 'internal',
+  };
+}
+
+export function validateLicense(): void {}
+
+export function setLicenseCheckSkipCondition(): void {}
+
+export default {
+  validateLicense,
+};
```

Comment:
```suggestion
export default {
  validateLicense(): void {}
};
```

Reply:
I guess all other exports are used only in tests

Reply:
In my opinion, it is better to leave exports the same regardless its usage

Reply:
OK

I've added testing for equality of modules signatures